swagger.json

```diff
@@ -6855,15 +6855,30 @@
                             "items": {
                                 "type": "object",
                                 "properties": {
-                                    "tagName": {
+                                    "tagId": {
+                                        "type": "number",
+                                        "format": "int64",
+                                        "description": "The id of the tag to grant this user access to.",
+                                        "example": "123"
+                                    },
+                                    "roleName" : {
                                         "type": "string",
-                                        "description": "The name of the tag to grant this user access to.",
-                                        "example": "North Carolina Distribution"
+                                        "description": "The name of the standard role type (case insensitive). Either this field or customRoleUuid must be provided.",
+                                        "example": "Read-Only Admin",
+                                        "enum": [
```

Comment:
I like that this approach provides easy access to built-in roles. However, if a user already has to make a `/GET` call for any custom role values, should we simply include standard roles in that list and accept just an `id` for both standard and custom here?

You may have already discussed with with @silverspace so feel free to just respond with "We already discussed and landed on this approach" 😄

Reply:
> Do we have a reserved name validation on custom roles, so customers cannot select standard role names like "Full Admin"?

customers cannot change or create roles that match the names of existing standard roles

Reply:
with generated sdk type things and for our api documentation in general, do we want to allow for this kind of "ids can be either strings or numbers" behavior?

Reply:
I might offer that we _could_ hardcode uuids for our existing role types (rather than use number ids) so that the api is consistent

Reply:
Assuming we create a `roles/` endpoint, might we change it to instead:
```
{
  "name": "Jane Doe",
  "email": "user@company.com",
  "authType": "basic",
  "organizationRole": "Read-Only Admin",
  "tagRoles": [
    {
      "tagId": "123",
      "roleId": "1"
    },
    {
      "tagId": "123",
      "roleId": "aaaa-aaaa-aaa-aaa"
    },
  ]
}
```

Reply:
Yes I am fine with treating the id as a string. I like this over creating a `uuid` for standard roles just for API.

Reply:
sounds good. I'll change this to use only the `roleId` instead of using `roleName`

Reply:
adjusted